src/Symfony/Component/Mailer/Mailer.php

```diff
@@ -24,21 +23,32 @@
  */
 class Mailer implements MailerInterface
 {
-    private $transport;
+    private $transports;
     private $bus;
     private $dispatcher;
 
-    public function __construct(TransportInterface $transport, MessageBusInterface $bus = null, EventDispatcherInterface $dispatcher = null)
+    public function __construct(Transports $transports, MessageBusInterface $bus = null, EventDispatcherInterface $dispatcher = null)
```

Comment:
Usually this kind of services named as `TransportLocator`

Reply:
As you know, I'm not a big fan of using a pattern name in a class name. `Transports` sounds much better to me.